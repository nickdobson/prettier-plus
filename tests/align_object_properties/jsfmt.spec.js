run_spec(__dirname, [ 'babel' ], { alignObjectProperties: true });