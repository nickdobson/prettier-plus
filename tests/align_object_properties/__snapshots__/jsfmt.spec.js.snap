// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`o.js 1`] = `
====================================options=====================================
alignObjectProperties: true
parsers: ["babel"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
o = {
  [name]: 'value',
  2:3,
  "k-2":2,
  keyasdf:3,
  test: {
      with: "lines"
  },
  asdf:3,
  key:1,
  [key_long]: 'value',
  tests: () => 
  "super long string to force a new line because I think that won't work like I want it to",
  as:3,test: {
    with: "lines"
}
};

=====================================output=====================================
o = {
  [name]  : "value",
  2       : 3,
  "k-2"   : 2,
  keyasdf : 3,
  test    : {
    with : "lines"
  },
  asdf       : 3,
  key        : 1,
  [key_long] : "value",
  tests      : () =>
    "super long string to force a new line because I think that won't work like I want it to",
  as   : 3,
  test : {
    with : "lines"
  }
};

================================================================================
`;
